examples/dos-games/zarf.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   description: Simple example to load classic DOS games into K8s in the airgap
   version: 1.2.0
 
+# The documentation key can be utilized to pass information to package consumers
```

Comment:
Does this addition warrant bumping the package version (and possibly publishing)?